website/docs/AutoCompile.md

```diff
@@ -0,0 +1,36 @@
+---
+id: autocompile
+title: Auto-compiling
```

Comment:
I wonder if it makes more sense to just mention these auto compile options in the Babel/TypeScript pages rather than having a dedicated page for it.

Reply:
Since it's dedicated behavior I felt a sub-page under Test Runner section was a good fit. Especially in light of https://github.com/TypeStrong/ts-node/issues/409, but seeing as WebdriverIO only supports Babel and TypeScript and there are pages for them already I see it makes sense to consolidate the information in both of those pages and remove this one which just effectively will point to either of them.